pkg/kudoctl/packages/verifier/template/verify_namespace.go

```diff
@@ -0,0 +1,65 @@
+package template
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/kudobuilder/kudo/pkg/engine/renderer"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/verifier"
+)
+
+var _ packages.Verifier = &NamespaceVerifier{}
+
+type NamespaceVerifier struct{}
+
+func (n NamespaceVerifier) Verify(pf *packages.Files) verifier.Result {
+	res := verifier.NewResult()
+
+	n.namespaceKindNotUsed(pf, &res)
+	n.checkNamespaceManifest(pf, &res)
+
+	return res
+}
+
+func (n NamespaceVerifier) checkNamespaceManifest(pf *packages.Files, res *verifier.Result) {
+	if pf.Operator.NamespaceManifest != "" {
+		val, ok := pf.Templates[pf.Operator.NamespaceManifest]
+		if !ok {
+			res.AddErrors(fmt.Sprintf("NamespaceManifest %q not found in /templates folder", pf.Operator.NamespaceManifest))
+			return
+		}
+		ns, err := renderer.YamlToObject(val)
+		if err != nil {
+			res.AddErrors(fmt.Sprintf("Unable to marshal NamespaceManifest %q ", pf.Operator.NamespaceManifest))
```

Comment:
```suggestion
			res.AddErrors(fmt.Sprintf("Unable to marshal NamespaceManifest %q ", pf.Operator.NamespaceManifest))
			return
```

We probably want to abort checking here, right?

Reply:
agree!  thank you!